requirements.txt

```diff
@@ -54,3 +52,24 @@ xlrd==1.2.0  # For panda's read_excel method
 numexpr==2.7.1
 
 backports.cached-property==1.0.0.post2
+
+
+
```

Comment:
Maybe add a comment saying these are for dash?

Reply:
Done in cf896f83